app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -285,6 +295,11 @@ class BrowserTabViewModel(
         )
 
         if (duckDuckGoUrlDetector.isDuckDuckGoQueryUrl(url)) {
+
+            newViewState = newViewState.copy(
+                    omnibarText = duckDuckGoUrlDetector.extractQuery(url) ?: "",
```

Comment:
Why update the omnibar text again? It is already happening above `omnibarTextForUrl(url)`

Reply:
`omnibarTextForUrl(url)` must have come in from a recent merge, done after i'd made that change above. Will remove the second query extraction.